### ISSUE TITLE
Is timeoff-management usable with Node v16? 

### ISSUE THREAD

**Issue by saschafoerster** [timeoff-management/timeoff-management-application#528]
@vpp I also had issues with Node 16 like #489, but I would to upgrade soon, so I can use other node apps on the same machine. Did you test 16 already? I will send more logs later on, but maybe you already have checked it yourself. :) Thank you!

**Comment by CharlemagneLasse** [comment]
No, this software only works correctly with nodejs 13 - so even with 14, the migration of databases fails (but the rest seems to work fine). Yes, nodejs 13 is completely unsupported. And it also needs python2 (for dependencies) - which is also completely unsupported since 2020. And the dependencies which this tool uses have numerous known security vulnerabilities. So it basically a no go to install this thing in any production environment nowadays. Which is rather bad because the actual software does a lot of things right (and simple).

@vpp are there any plans to get the dependency stuff fixed and migrate to any supported nodejs version? At the moment, only nodejs 18 and 20 are actively supported by nodejs upstream. 16 still receives minimal (maintenance) support.
For python, you would need at least support for python3.11 (and everything lower than 3.7 is completely unsupported).
